content/modules/redisearch/commands.md

```diff
@@ -10,30 +10,34 @@ categories: ["Modules"]
-| [FT.SYNDUMP](https://redis.io/commands/ft.syndump/) | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Supported">&#x2705; Free & Fixed</nobr></span> | Returns a list of synonym terms and their synonym group IDs. |
-| [FT.SYNUPDATE](https://redis.io/commands/ft.synupdate/) | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Supported">&#x2705; Free & Fixed</nobr></span> | Creates or updates a synonym group with additional terms. |
-| [FT.TAGVALS](https://redis.io/commands/ft.tagvals/) | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Supported">&#x2705; Free & Fixed</nobr></span> | Returns all distinct values indexed in a tag field. |
+| Command | Redis Enterprise Software | Redis Cloud<br />Flexible & Annual | Redis Cloud<br />Free & Fixed | Description |
+|:--------|:----------------------|:-----------------|:-----------------|:------|
+| [FT.AGGREGATE](https://redis.io/commands/ft.aggregate/) | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</nobr></span> | Runs a search query on an index and groups, sorts, transforms, limits, and/or filters the results. |
+| [FT.ALIASADD](https://redis.io/commands/ft.aliasadd/) | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</nobr></span> | Adds an alias to an index.  |
+| [FT.ALIASDEL](https://redis.io/commands/ft.aliasdel/) | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</nobr></span> | Removes an alias from an index. |
+| [FT.ALIASUPDATE](https://redis.io/commands/ft.aliasupdate/) | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</nobr></span> | Adds an alias to an index. If the alias already exists for a different index, it updates the alias to point to the specified index instead. |
+| [FT.ALTER](https://redis.io/commands/ft.alter/) | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</span> | <span title="Supported">&#x2705; Supported</nobr></span> | Adds a new field to an index. |
+| [FT.CONFIG GET](https://redis.io/commands/ft.config-get/) |  <span title="Not supported"><nobr>&#x274c; Not supported</span> | <span title="Not supported"><nobr>&#x274c; Not supported</span> | <span title="Not supported"><nobr>&#x274c; Not supported</nobr></span> | Displays configuration options. |
```

Comment:
`FT.CONFIG GET` on `Redis Cloud Flexible & Annual` - a customer can contact support to query about his current setting.

We can reuse note 2 and change the wording to 'query about the current value or to request a change ... '

Reply:
This info has been added. Thanks!